spec/message_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'message'
+
+
+describe Message do
+  it 'Sends a sms confirmation message with 2 arguments(number,name)' do
+    expect(Message.new.send("+555","Jim"))
+      .to eq("Sent message to Jim")
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)